browser/src/map/Map.js

```diff
@@ -989,6 +989,13 @@ L.Map = L.Evented.extend({
 		return this.options.crs.pointToLatLng(L.point(point), zoom);
 	},
 
+	/// Get LatLng coordinates after negating the X cartesian-coordinate.
```

Comment:
This describes what the function does, which is easy to read/see, but it doesn't explain why or when/where this is useful. It would be helpful to maintainers to read a note as to when this is typically used so they can more easily understand the purpose of this function. I.e. the 'why' as opposed to the 'what'.